src/socketHandlers/index.js

```diff
@@ -0,0 +1,4 @@
+import io from 'socket.io-client';
+const socket = io('localhost:9000/game');
```

Comment:
I think web socket doesn't work on Heroku. because on Heroku, host of websocket server is not `localhost:9000`